builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -620,7 +685,9 @@ def validateDeploymentConfigurations(def prop) {
 		} else if (Integer.parseInt(prop['providerTimeout']) > 300) { // Should not be a high
 			error "Wrong configuration. Value for Key 'providerTimeout' should be a less than 160"
 		}
-	} else {
+	} else if (config['platform'] == "azure") {
```

Comment:
Why would we check for azure related stuff in this function (that handles s3 related errors)?

Reply:
This method is existing and it checks if timeout value > 300 from deployment-env.yml
and this is aws thing.  Azure function has default for timeout and memory size so we use default not make it configurable. 
providerRuntime : nodejs8.10
providerMemorySize : 256
providerTimeout : 160